open_feature/hooks/hook_context.py

```diff
@@ -0,0 +1,20 @@
+from open_feature.evaluation_context.evaluation_context import EvaluationContext
+from open_feature.flag_evaluation.flag_type import FlagType
+
+
+class HookContext:
```

Comment:
Could this be simplified to a python `dataclass` ?

Reply:
Yes agreed, only supported in python 3.7+ so if we decide to go below that we'd have to remove it